lib/excon/ssl_socket.rb

```diff
@@ -4,6 +4,8 @@ class SSLSocket < Socket
       OpenSSL::SSL::SSLSocket.public_method_defined?(m)
     end
 
+    VERIFY_ENV_VAR = 'EXCON_SSL_VERIFY_PEER'
```

Comment:
Maybe just for clarity/explicitness this could be `VERIFY_PEER_ENV_VAR` for the constant. We might also want to consider putting this into `lib/excon/constants` as that is where most all of the others live.